### ISSUE TITLE
Execution issue in MAC due to  non-unix line ending

### ISSUE THREAD

**Issue by ashishpatel1992** [elderfo/react-native-storybook-loader#101]
There is an error while executing `rnstl` due to  non-UNIX line endings 
```sh
yarn storybook-native
yarn run v1.22.10
$ rnstl
env: node\r: No such file or directory
error Command failed with exit code 127.
```

Fix:
```sh
dos2unix ./node_modules/react-native-storybook-loader/out/rnstl-cli.js
```

**Comment by aymkin** [comment]
Solution on WebStorm
* open file `node_modules/.bin/rnstl`
* make sure the IDE displays line separator 

![image](https://user-images.githubusercontent.com/10321850/116694770-5afcc380-a9c8-11eb-9c8a-09b168a0abf1.png)

* select `LF`

![image](https://user-images.githubusercontent.com/10321850/116694884-8089cd00-a9c8-11eb-9ead-f9b8f05772f3.png)

run script again

**Comment by christophemenager** [comment]
I confirm I have the same issue ! 
I used the solution given by @ashishpatel1992 and it works 🔥 Thanks a lot

**Comment by elenitaex5** [comment]
Thanks @ashishpatel1992 . I had same issue too and your fix works fine!!! 🥳
You rock! 🤘

**Comment by elderfo** [comment]
I published v2.0.4 using LF for newlines. I also changed the default to LF so when I build from my Windows machine instead of my mac this doesn't happen again.
